ejb3/src/main/java/org/jboss/as/ejb3/timerservice/persistence/filestore/LegacyFileStore.java

```diff
@@ -1,15 +1,6 @@
 package org.jboss.as.ejb3.timerservice.persistence.filestore;
```

Comment:
Not related to this PR really but if this is line 1 is this class missing a copyright header?

Reply:
I'll add a copyright header to this file.